main/src/kotlinx/team/infra/Publishing.kt

```diff
@@ -145,7 +145,7 @@ private fun Project.createBuildRepository(name: String, rootBuildLocal: Task) {
                 delete(dir)
             }
 
-            tasks.withType(PublishToMavenRepository::class.java).configureEach {
+            tasks.withType(PublishToMavenRepository::class.java) {
```

Comment:
I see the problem, but I believe it should be possible to solve it without defying the laziness of the configuration, somehow like
```
compositeTask.dependsOn(tasks.withType(PublishToMavenRepository::class.java).matching { predicate })
```

Reply:
Done